packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -48,11 +48,6 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
      */
     uint256 internal constant FINALIZE_GAS_BUFFER = 20_000;
 
-    /**
-     * @notice Minimum time (in seconds) that must elapse before a withdrawal can be finalized.
-     */
-    uint256 public immutable FINALIZATION_PERIOD_SECONDS;
```

Comment:
Technically this breaks the API but I'm OK with that because we're not on mainnet yet.